File: lab/python/fatorial/fatorial3.py
def fatorial_recursivo(n):
    if n == 1 or n == 0:
        print(f"{n} =", end=" ")
        return 1
    print(f"{n} x", end=" ")
    return n * fatorial_recursivo(n-1)


print(fatorial_recursivo(5))

File: lab/javascript/fatorial/fatorial3.js
function fatorial_recursivo(n) {
  if (n === 1 || n === 0) {
    process.stdout.write(`${n} = `);
    return 1;
  }
  process.stdout.write(`${n} x `);
  return n * fatorial_recursivo(n - 1);
}

// Teste
console.log(fatorial_recursivo(5));
